app/layout/scenarios/sidebar/tabs/component.tsx

```diff
@@ -38,7 +38,6 @@ export const ScenariosSidebarTabs: React.FC<ScenariosSidebarTabsProps> = () => {
   const dispatch = useDispatch();
 
   useEffect(() => {
-    console.info('---> HERE', refTab, refSubtab);
     dispatch(setTab(refTab));
```

Comment:
i was not able to save solution-details sub tab. For any reason its been cleaned from store